generators/app/templates/src/services/AuthServices.ts

```diff
@@ -0,0 +1,43 @@
+import { ApiResponse } from 'apisauce';
+
+import { User, Credentials } from '~app/contexts/UserContext/reducer';
+
+import api from '../config/api';
+
+import LocalStorageService from './LocalStorageService';
+
+const TOKEN_FIELD_NAME = 'sessionToken';
+
+export interface LoginError {
+  message: string;
+}
+
+export const setCurrentUser = (currentUser: User) => {
+  api.setHeader('Authorization', currentUser.sessionToken);
+  LocalStorageService.setValue(TOKEN_FIELD_NAME, currentUser.sessionToken);
+};
+
+export const getCurrentUser = () => {
+  const currentSessionToken = LocalStorageService.getValue(TOKEN_FIELD_NAME);
+  if (currentSessionToken) {
+    api.setHeader('Authorization', currentSessionToken);
```

Comment:
I'm not sure if we should do this here. I realize you only execute this function when the app is loaded but we might want to do it elsewhere without setting the header. I think I'd split it, or maybe just take the api part and do that in the component

Reply:
We are actually setting it on the `setCurrentUser` function which is used in the component, so I'll delete it from here. However, it seems logical to me that when we set the current user we set the corresponding header to the api.